### ISSUE TITLE
VPC RVR: Delete first tier with Private GW - Both VRs become MASTER

### ISSUE THREAD

**Issue by richardlawley** [apache/cloudstack#3402]
##### ISSUE TYPE

 * Bug Report

##### COMPONENT NAME

~~~
VPC VR
~~~

##### CLOUDSTACK VERSION

~~~
4.11.2+
~~~

##### CONFIGURATION

Redundant VPC

##### OS / ENVIRONMENT

N/A

##### SUMMARY

I'm creating this bug report to track [CLOUDSTACK-9885](https://issues.apache.org/jira/browse/CLOUDSTACK-9885), since there's nothing in any milestones over here for it.  A PR #2128 was created, but the PR wasn't completed an alternative solution proposed.  #2719 attempted to address the general situation, but missed the specifics of needing to delete a tier.

Short version - VPC with Deleted Tier and Private Gateway may have keepalived configured on the private gateway, leading to both VRs in MASTER.

##### STEPS TO REPRODUCE

1. Create Redundant VPC, configure a Tier 1 (+deploy VM)
2. Add Private Gateway
3. Add Tier 2 (+deploy VM)
4. Delete Tier 1

##### EXPECTED RESULTS

Keepalived should be configured on the interface for Tier 2

##### ACTUAL RESULTS

Keepalived was configured on the interface for Private Gateway (first interface with traffic type = Guest)

**Comment by rhtyd** [comment]
Related https://github.com/apache/cloudstack/pull/2128

**Comment by rhtyd** [comment]
This exists since 4.9, moving to next minor milestone.

**Comment by DaanHoogland** [comment]
possible fix out so putting back to 4.13.1 for now

**Comment by DaanHoogland** [comment]
merged on 4.13
